src/controllers/artist.js

```diff
@@ -0,0 +1,110 @@
+// exports.create = (req, res) => {
+//     res.sendStatus(201);
+// }
+// const create = (req, res) => {
+//     res.sendStatus(201);
+// }
+
+// module.exports = { create }
+const db = require('../db/index');
+
+exports.createArtist = async (req, res) => {
+    const { name, genre } = req.body
+  
+    try {
+      const { rows: [ artist ] } = await db.query(`INSERT INTO Artists (name, genre) VALUES ($1, $2) RETURNING *`, [name, genre])
+      res.status(201).json(artist)
+    } catch (err) {
+      res.status(500).json(err.message)
+    }
+  }
+
+  exports.getAllArtists = async (_, res) => {
+    try {
+      const { rows } = await db.query('SELECT * FROM Artists')
+      res.status(200).json(rows)
+    } catch (err) {
+      res.status(500).json(err.message)
+    }
+  }
+
+  exports.getArtistById = async (req, res) => {
+    try {
+      const { id } = req.params
+      const { rows: [ artist ] } = await db.query('SELECT * FROM Artists WHERE id = $1', [ id ])
```

Comment:
Long lines like this are a great illustration of why its important to format your code with prettier. Compare it with this:
```
    const {
      rows: [artist],
    } = await db.query('SELECT * FROM Artists WHERE id = $1', [id]);
```
Making things fit more vertically instead of horizontally like this makes it easier to read and less tiring for the eyes